### ISSUE TITLE
the proper way to perform the parse_tree

### ISSUE THREAD

**Issue by shmilyhh** [EmilStenstrom/conllu#6]
When I use the parse_tree() to get the relationship between each term, it can not show the same format as in your example, instead the output is in one line. Can you help me to get the same format as in your example of parse_tree()? Thanks.

**Comment by EmilStenstrom** [comment]
Hi @shmilyhh. In my example I have manually added linebreaks to make the output easier to read. Doing this automatically wouldn't work, unless I changed the repr() of my TreeNode to something else.

But I'm guessing what you are looking for is a way to *print* the tree structure. For that, there is a method called print_tree. I've added an example in the bottom of the [README ](https://github.com/EmilStenstrom/conllu/blob/master/README.md) file. Is that what you're looking for?
